guides/v2.3/graphql/payment-methods/authorize-net.md

```diff
@@ -0,0 +1,87 @@
+---
+group: graphql
+title: Authorize.Net payment method
+---
+
+Accept.js is a JavaScript-based solution for sending secure payment data directly to Authorize.Net. The Accept JavaScript library intercepts the payment data before it is passed to Magento and submits it directly to Authorize.Net, which replaces it with a one-time-use token, or payment nonce. This payment nonce, which is returned by the JavaScript library, is used in place of payment data.
```

Comment:
For some reason, starting the sentence off with the JS filename seems odd. Perhaps:
```suggestion
The Accept JavaScript library is a JavaScript-based solution for sending secure payment data directly to Authorize.Net. This JavaScript library intercepts the payment data before it is passed to Magento and submits it directly to Authorize.Net, which replaces it with a one-time-use token, or payment nonce. This payment nonce, which is returned by the JavaScript library, is used in place of payment data.
```

Reply:
Keeping this as is, because Accept.js is the official name.